saleor/graphql/decorators.py

```diff
@@ -0,0 +1,35 @@
+from functools import wraps
+
+import six
+from graphql_jwt import exceptions
+from graphql_jwt.decorators import context
+
+
+def user_passes_test(test_func):
```

Comment:
I think it could use a short docstring to explain what it does, the name doesn't tell us much 👍